uportal-war/src/main/java/org/jasig/portal/groups/pags/testers/PropertyRegexTester.java

```diff
@@ -36,10 +37,25 @@
 public class PropertyRegexTester extends RegexTester {
     private Logger logger = LoggerFactory.getLogger(this.getClass());
 
+    public PropertyRegexTester(IPersonAttributesGroupTestDefinition definition) {
+        super(definition);
+        final String propertyName = definition.getTestValue();
+        String regexExpression = PropertiesManager.getProperty
+                (propertyName, "");
+        if (StringUtils.isBlank(regexExpression)) {
+            logger.error("Unable to find property name {} in portal.properties or has empty value."
+                + " PAGS PropertyRegexTester will always return false for attribute {}", propertyName,
+                definition.getAttributeName());
+        }
+        setPattern(regexExpression);
+    }
```

Comment:
Same as PropertyInvertedRegexTester in regards to private method to avoid code duplication.